data/mships.txt

```diff
@@ -4412,7 +4412,7 @@ ship "SKVS-2 Mudskipper"
 	explode "small explosion" 30
 	explode "medium explosion" 20
 	"final explode" "final explosion small"
-	description `Small, lightly armed freighter for short-ranged shipping from earlier and more peaceful era. These are most often seen in more secure area of space as they lack of any effective defense against pirates nor the range to travel far beyond their homeport.`
+	description `A small, lightly armed freighter for short-ranged shipping from an earlier and more peaceful era. These are most often seen in more secure areas of space as they lack any effective defense against pirates and do not have the range to travel far beyond their home port.`
```

Comment:
```suggestion
	description `Small, lightly armed freighter for short-ranged shipping from an earlier and more peaceful era. These are most often seen in more secure areas of space as they lack any effective defense against pirates nor the range to travel far beyond their homeport.`
```
I don't like this more verbose phrasing, "nor" works fine. Also "homeport" is a word.